timesketch/frontend/src/components/Common/UploadForm.vue

```diff
@@ -638,11 +663,14 @@ export default {
       if (this.form.file.size <= 0) {
         this.error.push("Please select a non empty file");
       }
-      let allowedExtensions = ["csv", "json", "jsonl", "plaso"];
```

Comment:
hm I am not sure if we should take away json as a filetype. Yes in theory everyone should use jsonl as extension but I am not sure it might break stuff for people